ironfish/src/memPool/memPool.test.ts

```diff
@@ -22,7 +21,7 @@ import { mempoolEntryComparator } from './memPool'
 // but not the mempool
 async function createTransactions(
   node: IronfishNode,
-  from: SpendingAccount,
+  from: Account,
```

Comment:
can we rebase and get rid of this somehow?

Reply:
yes, i'll fix this issue in the dst branch